carma-messenger-core/cpp_message/src/cpp_message.cpp

```diff
@@ -1187,33 +1247,14 @@ namespace cpp_message
                 output_geometry->heading = msg_geometry.heading;
                 
                 // nodes
-                auto nodes_len = msg_geometry.nodes.size();
-                TrafficControlGeometry::TrafficControlGeometry__nodes* nodes_list;
-                nodes_list = (TrafficControlGeometry::TrafficControlGeometry__nodes*) calloc(1, sizeof(TrafficControlGeometry::TrafficControlGeometry__nodes));
-                
+                // auto nodes_len = msg_geometry.nodes.size();
```

Comment:
Needed?

Reply:
This was moved out of the if blocks, to line 1030, so that it can be freed at the end

The comment is unnecessary